runtime/westend/constants/src/lib.rs

```diff
@@ -96,6 +96,14 @@ pub mod fee {
 	}
 }
 
+/// System Parachains.
+pub mod system_parachain {
+	/// Westmint parachain ID.
+	pub const WESTMINT_ID: u32 = 1000;
+	/// Collectives parachain ID.
+	pub const COLLECTIVES_ID: u32 = 1001;
```

Comment:
For consistency we should add: 
```suggestion
	pub const COLLECTIVES_ID: u32 = 1001;
	use xcm::latest::prelude::*;

	frame_support::match_types! {
		pub type SystemParachains: impl Contains<MultiLocation> = {
			MultiLocation { parents: 0, interior: X1(Parachain(WESTMINT_ID | COLLECTIVES_ID)) }
		};
	}
```